src/Core/Impl/Extensions/StringExtensions.cs

```diff
@@ -146,9 +146,6 @@ public static string AsQuotedArguments(this IEnumerable<string> args)
             return "\"{0}\"".FormatInvariant(arg);
         }
 
-        public static bool PathStartsWith(this string s, string prefix)
```

Comment:
Should we have it instead calling `PathUtils.PathStartsWith` ?

Reply:
We have a bit of a mess in our extensions. From my understanding, `StringExtensions` should have things that are applicable to any strings, `PathUtils` should have string operations that are related to paths only, and `FileSystemExtensions` should have things like `FindFile`.